modules/abstract-eth/src/abstractEthLikeNewCoins.ts

```diff
@@ -1362,6 +1362,9 @@ export abstract class AbstractEthLikeNewCoins extends AbstractEthLikeCoin {
 
     // get balance of wallet
     const txAmount = await this.queryAddressBalance(params.walletContractAddress);
+    if (txAmount.toString(10) === '0') {
```

Comment:
I would use bignumber just in case and also check its equal or lesser than 0
```suggestion
    if (new BigNumber(txAmount).lte(0)) {
```

Reply:
done